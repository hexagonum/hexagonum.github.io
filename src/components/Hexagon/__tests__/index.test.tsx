import { render } from '@testing-library/react';
import { Hexagon } from '..';

jest.mock('next/router', () => ({
  useRouter: jest.fn().mockReturnValue({ query: { visible: 'true' } }),
}));

describe('Hexagon', () => {
  it('to match snapshot', () => {
    const { container } = render(<Hexagon />);
    expect(container).toMatchSnapshot();
  });
});
