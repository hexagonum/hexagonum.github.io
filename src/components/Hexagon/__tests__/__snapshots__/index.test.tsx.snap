// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hexagon to match snapshot 1`] = `
<div>
  <svg
    height="112"
    style="pointer-events: none;"
    version="1.1"
    viewBox="0 0 128 110.85125168440814"
    width="128"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0 55.42562584220407L32 0L96 0L128 55.42562584220407L96 110.85125168440814L32 110.85125168440814Z"
      stroke="#ffffff"
      stroke-width="5"
      style="pointer-events: none;"
    />
  </svg>
</div>
`;
